website/src/pages/use-cases/code-reuse.tsx

```diff
@@ -0,0 +1,313 @@
+                            <p>
+                                “If I’m developing code for a library that might draw charts, for example, we don’t want
+                                30 different ways to draw a chart at FactSet. With Sourcegraph, I can search the code to
+                                find other chart examples, and simply copy the code. This saves us time and ensures
+                                consistency.”
+                            </p>
+                            <figcaption className="pt-2 text-muted text-center">
+                                &mdash; Joseph Majesky, Software Engineer at FactSet
+                            </figcaption>
+                        </section>
+                        <img className="my-4" src="/external-logos/factset-logo.svg" width="150" alt="Factset logo" />
+                        <Link
+                            className="d-block font-weight-bold"
+                            to="/case-studies/factset-migrates-from-perforce-to-github"
+                        >
+                            Read the case study
+                            <ArrowRightIcon className="ml-2" />
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-bold text-md-center text-left mb-lg-6 mb-md-6">How Sourcegraph helps</h1>
+                </div>
+                <div className="pb-lg-5 pb-md-6 pb-5">
+                    <CustomCarousel items={items} autoAdvance={true} smallPanel={true} />
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-light-gray-3">
+            <ContentSection>
+                <div className="row d-flex flex-column mx-4 mx-lg-0 py-7 align-items-lg-center align-items-left">
+                    <div className="mb-5 d-flex flex-column text-start text-md-center max-w-600 mx-auto">
+                        <h1 className="font-weight-bold">Get started with Sourcegraph</h1>
+                        <p>
+                            Make it easier to discover trustworthy code for reuse so your teams can spend more time
+                            solving new problems and less time rewriting code.
+                        </p>
+                    </div>
+                    <div className="text-center col-12 px-0">
+                        <Link
+                            className="btn btn-primary col-12 col-md-4 col-lg-2 min-w-200"
+                            to="/demo"
+                            title="Request a Demo."
+                        >
+                            Request a demo
+                        </Link>
```

Comment:
Hey @st0nebraker! Marketing noted in the https://github.com/sourcegraph/about/pull/5274 QA doc that the "Explore other use cases" link should [appear below](https://docs.google.com/document/d/1Khtt6oV1eQ3R38OurNGtew04NN2XaOPNO2GW391uDwI/edit?disco=AAAAYs8yjuM) the "Request a demo" CTA, although it doesn't appear in the Code Health or Code Reuse Figma files. Just wanted to pass it along here. This is how it appears in the other Use Case pages:
```
<Link to="/use-cases" className="d-flex justify-content-center mt-4">
    <p className="font-weight-bold">Explore other use cases</p>
</Link>
```
Figma:
<img width="500" alt="Screen Shot 2022-04-28 at 3 35 58 PM" src="https://user-images.githubusercontent.com/60713139/165841207-8c7289be-bbbb-4e79-bc8d-6d69c1c96bd2.png">

Reply:
Thanks, Tim! Added it in. We might encounter some merge conflicts when merging the 3 Use Case branches in soon 😬  Just to note and beware of

Reply:
Roger! We can coordinate on the order. fyi, I am [waiting on non-blocking feedback](https://github.com/sourcegraph/about/pull/5297#issuecomment-1111392132) to merge in Incident Response #5259, and Code Health #5274 is [still in QA](https://github.com/sourcegraph/about/pull/5274#issuecomment-1112604002).

Reply:
Cool, sounds good! I left another comment about unifying the use cases inconsistencies, we can make another story to capture this extra effort and manage the scope creep too!